src/config/products.js

```diff
@@ -0,0 +1,5 @@
+import FIREFOX_DESKTOP from './firefox-desktop';
+
+export default {
+  firefoxDesktop: FIREFOX_DESKTOP,
```

Comment:
Is the idea that the product dropdown will have `Object.keys(PRODUCTS)`?

Reply:
Yes, & that suggests some additions to each per-product config file once we get to that